src/pages/SearchPage.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from '../components/Button'
+import CropItem from '../components/CropItem'
+import Header from '../components/Header'
+
+SearchPage.propTypes = {
+  crops: PropTypes.array.isRequired,
+  onClickDetails: PropTypes.func.isRequired,
+  onClickFavorites: PropTypes.func.isRequired,
+}
+
+export default function SearchPage({
+  crops,
+  onClickFavorites,
+  onClickDetails,
+}) {
+  const [searchTerm, setSearchTerm] = useState('')
+  const [searchResults, setSearchResults] = useState([])
+
+  return (
+    <Wrapper>
+      <Header>Search</Header>
+      <Input
+        placeholder="Search your favorite crops  🔍"
+        name="Search"
+        value={searchTerm}
+        onChange={handleChange}
+      />
+      <Output>
+        {searchTerm.length === 0 ? (
```

Comment:
I think if you set the initial state of `searchResults` to `crops` you don't need the conditional rendering here
line 20:
```
const [searchResults, setSearchResults] = useState(crops)
```

Reply:
Great Thanks! it worked:) makes the code a lot nicer